### PR TITLE
Active directory docs

### PR DESCRIPTION
**Pull request by myoungerman**

### CODE REVIEW

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md [hunk 1]
```diff
@@ -34,7 +38,9 @@ This tutorial will show you how to configure SCIM (System for Cross-Domain Ident
 
       ![App registration tab](images/azure-ad-application-and-openid/app-registration.png)
 
-1. Select the **All applications** tab. Your newly created app should appear in the **Display name** list.
+1. Select the **All applications** tab.
+
+      Your newly created app should appear in the **Display name** list.
```
Comment (anetroger) [inline src/docs/tutorials/pages/azure-ad-application-and-openid.md:43]: Delete 'should'. Present tense is preferred.

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md [hunk 2]
```diff
@@ -44,7 +50,7 @@ This tutorial will show you how to configure SCIM (System for Cross-Domain Ident
 
       ![The application ID](images/azure-ad-application-and-openid/application-id.png)
 
-1. Select the **Endpoints** tab. The **Endpoints** panel will appear.
+1. Select the **Endpoints** tab. The **Endpoints** panel appears.
```
Comment (anetroger) [inline src/docs/tutorials/pages/azure-ad-application-and-openid.md:53]: Separate line for step results

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md [hunk 3]
```diff
@@ -54,7 +60,7 @@ This tutorial will show you how to configure SCIM (System for Cross-Domain Ident
 
       ![Certificates and secrets tab](images/azure-ad-application-and-openid/certificates-tab.png)
 
-1. Select **+ New Client Secret**. An **Add a client secret** panel will appear.
+1. Select **+ New Client Secret**. An **Add a client secret** panel appears.
```
Comment (anetroger) [inline src/docs/tutorials/pages/azure-ad-application-and-openid.md:63]: Separate line for step results

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md [hunk 4]
```diff
@@ -203,7 +219,7 @@ This tutorial will show you how to configure SCIM (System for Cross-Domain Ident
 
 1. Select the **+ Add user/group** tab at the top of the center panel.
 
-1. Select **None Selected** under **Users**. The **Users** panel will appear on the right.
+1. Select **None Selected** under **Users**. The **Users** panel appears on the right.
```
Comment (anetroger) [inline src/docs/tutorials/pages/azure-ad-application-and-openid.md:222]: Separate line for step results - please review this file for all occurrences of step results.

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md [hunk 5]
```diff
@@ -248,9 +270,11 @@ https://drive.google.com/file/d/19DmYILxPMSrNshJiC95eRHLK59H0NBFp/view -->
 
       ![Get started button](images/azure-ad-application-and-openid/get-started.png)
 
-      The **Provisioning** page will load.
+      The **Provisioning** page loads.
```
Comment (anetroger) [inline src/docs/tutorials/pages/azure-ad-application-and-openid.md:273]: Be consistent - use 'appears' in all occurrences.

**Review by anetroger — CHANGES_REQUESTED** [review]
I have reviewed up to half of the file. Please, be consistent when you write step results and when you refer to UI terms. Delete all screenshots that can be written as step results and replace them by step results. Make sure that you put context first, as in the style guide. When you do that for all files, please resubmit the PR.

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
-# Setting up an Azure Active Directory Application and OpenID
+# Set up an Azure Active Directory Application and OpenID
 
 **APPROXIMATE READING TIME: 30 MINUTES**
 
-This tutorial will show you how to configure SCIM (System for Cross-Domain Identity Management) endpoints and create an aiWare organization, Azure Active Directory (AD) application, and users.
+This tutorial shows you how to configure SCIM (System for Cross-Domain Identity Management) endpoints and create an aiWare organization, Azure Active Directory (AD) application, and users.
```
[the remark this answers is not in the document]
Comment (myoungerman) [reply src/docs/tutorials/pages/azure-ad-application-and-openid.md]: fixed

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md [hunk 7]
```diff
@@ -16,11 +16,15 @@ This tutorial will show you how to configure SCIM (System for Cross-Domain Ident
...
+1. Select **+ Create your own application**.
+
+      The **Create your own application** panel appears.
 
 1. In the **Create your own application** panel, name your application and select the **Integrate any other application you don't find in the gallery** radio button.
 
-1. Select **Create**. The app overview page will load and show your app's properties.
+1. Select **Create**.
+
+      The app overview page loads and show your app's properties.
```
[the remark this answers is not in the document]
Comment (myoungerman) [reply src/docs/tutorials/pages/azure-ad-application-and-openid.md]: fixed

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md (hunk 1 above)
Comment (myoungerman) [reply src/docs/tutorials/pages/azure-ad-application-and-openid.md]: fixed

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md (hunk 2 above)
Comment (myoungerman) [reply src/docs/tutorials/pages/azure-ad-application-and-openid.md]: fixed

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md (hunk 3 above)
Comment (myoungerman) [reply src/docs/tutorials/pages/azure-ad-application-and-openid.md]: fixed

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md (hunk 4 above)
Comment (myoungerman) [reply src/docs/tutorials/pages/azure-ad-application-and-openid.md]: fixed

File: src/docs/tutorials/pages/azure-ad-application-and-openid.md (hunk 5 above)
Comment (myoungerman) [reply src/docs/tutorials/pages/azure-ad-application-and-openid.md]: fixed

**Comment by anetroger** [comment]
Approving this PR as per Mitchell request.

**Review by anetroger — APPROVED** [review]
Approving the PR without reviewing changes.

**Pull request merged**